src/wrapped_radix_engine_toolkit/high_level/manifest_building/manifests.rs

```diff
@@ -0,0 +1,290 @@
+};
+
+use radix_engine_common::prelude::NonFungibleLocalId as ScryptoNonFungibleLocalId;
+
+impl TransactionManifest {
+    pub fn faucet(
+        include_lock_fee_instruction: bool,
+        address_of_receiving_account: &AccountAddress,
+    ) -> Self {
+        let mut builder = ScryptoManifestBuilder::new();
+
+        if include_lock_fee_instruction {
+            builder = builder.lock_fee_from_faucet()
+        }
+
+        let scrypto_manifest = builder
+            .get_free_xrd_from_faucet()
+            .try_deposit_entire_worktop_or_abort(
+                address_of_receiving_account.scrypto(),
+                None,
+            )
+            .build();
+
+        TransactionManifest::from_scrypto(
+            scrypto_manifest,
+            address_of_receiving_account.network_id(),
+        )
+    }
+
+    pub fn marking_account_as_dapp_definition_type(
+        account_address: &AccountAddress,
+    ) -> Self {
+        Self::set_metadata(
+            account_address,
+            MetadataKey::AccountType,
+            MetadataValueStr::DappDefinition,
+        )
+    }
+
+    pub fn set_owner_keys_hashes(
+        address_of_account_or_persona: &AddressOfAccountOrPersona,
+        owner_key_hashes: Vec<PublicKeyHash>,
+    ) -> Self {
+        Self::set_metadata(
+            address_of_account_or_persona,
+            MetadataKey::OwnerKeys,
+            ScryptoMetadataValue::PublicKeyHashArray(
+                owner_key_hashes.into_iter().map(|h| h.into()).collect_vec(),
+            ),
+        )
+    }
+
+    fn account_withdraw_non_fungibles(
+        builder: ScryptoManifestBuilder,
+        owner: &AccountAddress,
+        resource_address: &ResourceAddress,
+        non_fungible_local_ids: &[NonFungibleLocalId],
+    ) -> ScryptoManifestBuilder {
+        builder.withdraw_non_fungibles_from_account(
+            owner,
+            resource_address,
+            non_fungible_local_ids
+                .iter()
+                .cloned()
+                .map(Into::<ScryptoNonFungibleLocalId>::into),
+        )
+    }
+
+    pub fn stake_claims(
+        owner: &AccountAddress,
+        stake_claims: Vec<StakeClaim>,
+    ) -> Self {
+        let account_address = owner;
+        let network_id = account_address.network_id();
+        let xrd_address = &ResourceAddress::xrd_on_network(network_id);
+
+        let mut builder = ScryptoManifestBuilder::new();
+
+        let bucket_factory = BucketFactory::default();
+
+        for stake_claim in stake_claims.iter() {
+            let claim_address = &stake_claim.resource_address;
+            let validator_address = &stake_claim.validator_address;
+
+            // Withdraw non fungibles from account
+            builder = Self::account_withdraw_non_fungibles(
+                builder,
+                account_address,
+                claim_address,
+                &stake_claim.ids,
+            );
+
+            let bucket = &bucket_factory.next();
```

Comment:
@iamyulong @0xOmarA here is building a stake claims manifest, using `&bucket_factory.next();`